satellite/accounting/tally/tally_test.go

```diff
@@ -161,54 +161,45 @@ func TestCalculateBucketAtRestData(t *testing.T) {
 		inline       bool
 		last         bool
 	}{
```

Comment:
This change here is intentional, because bucket information shouldn't be in pointer db anymore.